<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenshinImpactNaturalLawClass</class>
 <widget class="QMainWindow" name="GenshinImpactNaturalLawClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天理</string>
  </property>
  <property name="windowIcon">
   <iconset theme="MainIcon" resource="GenshinImpactNaturalLaw.qrc">
    <normaloff>:/icon/resource/icon/MainIcon.ico</normaloff>:/icon/resource/icon/MainIcon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="MainBackgroundImageRect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>730</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225, 225, 225)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="GenshinImpactNaturalLaw.qrc">:/BackgroundImage/resource/bg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabelRect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(20, 20, 20);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  天理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartGame">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>626</y>
      <width>238</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(255, 199, 9);
color:rgb(134, 77, 29);
border:none;
border-radius:3px;
}
QPushButton:hover {
background-color:rgb(255, 204, 33);
}
QPushButton:pressed {
background-color:rgb(255, 204, 33);
}</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TitleMin">
    <property name="geometry">
     <rect>
      <x>1211</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
border:none;
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin0.png)
}
QPushButton:hover {
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}
QPushButton:pressed {
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TitleSet">
    <property name="geometry">
     <rect>
      <x>1172</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(255, 0, 0);
border:none;
background-image:url(:/TitleButtonImage/resource/titlebutton/bset0.png)
}
QPushButton:hover {
background-image:url(:/TitleButtonImage/resource/titlebutton/bset1.png)
}
QPushButton:pressed {
background-image:url(:/TitleButtonImage/resource/titlebutton/bset1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TitleExit">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
border:none;
background-image:url(:/TitleButtonImage/resource/titlebutton/bexit0.png)
}
QPushButton:hover {
background-image:url(:/TitleButtonImage/resource/titlebutton/bexit1.png)
}
QPushButton:pressed {
background-image:url(:/TitleButtonImage/resource/titlebutton/bexit1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="GrayBackGround_Right_Rect">
    <property name="geometry">
     <rect>
      <x>1204</x>
      <y>30</y>
      <width>76</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 38);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_1">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>66</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/Bilibili_icon.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_2">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>136</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/QQ_icon.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_3">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>206</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/GitHub_icon.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_4">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>276</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Linke_5">
    <property name="geometry">
     <rect>
      <x>1222</x>
      <y>346</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(0,0,0,120);
color:rgb(255, 255, 255);
border:rgb(82, 92, 115);
border-radius:20px;
border-image:url(:/LinkeIcon/resource/LinkeIcon/KYJG_yuanshen.png);
}
QPushButton:hover {
border:2px solid rgb(255, 210, 132);
}
QPushButton:pressed {
border:2px solid rgb(255, 210, 132);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Page_Rect">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>234</y>
      <width>460</width>
      <height>213</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>63</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(200, 200, 200,50);
border:1px solid rgb(248, 250, 251);
border-radius:0px;
color:rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>463</y>
      <width>460</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
    font-family:&quot;微软雅黑&quot;;
	font-size:20px;
	color:rgb(255, 255, 255);
	background-color:rgba(0,0, 0,100);
}

QTabWidget::pane
{
	border:none;
}

QTabBar::tab{
 	margin-top: 0px; 
	margin-left: 0px; 
	width:69px;
	height:36px;
	color:rgb(255, 255, 255);
	border:none;
}
 
QTabBar::tab:selected
{
	width:69px;
	height:36px;
	color:rgb(255, 210, 132);
	border:none;
	border-bottom:2px solid rgb(255, 210, 132);
}
 QTabBar::tab:hover
{
	width:69px;
	height:36px;
	color:rgb(255, 210, 132);
	border:none;
	border-bottom:2px solid rgb(255, 210, 132);
}
 
QTabWidget::tab-bar {
	border:none;
	left:10px;
} 
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0, 0,0);</string>
     </property>
     <attribute name="title">
      <string>帮助</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>421</width>
        <height>66</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_Page_1">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 font-family:&quot;微软雅黑&quot;;
	font-size:16px;
text-align: left;
	color:rgb(255, 255, 255);
}
QPushButton:hover{
   font-family:&quot;微软雅黑&quot;;
	font-size:16px;
	color:rgb(255, 210, 132);
}
QPushButton:pressed{
    font-family:&quot;微软雅黑&quot;;
	font-size:16px;
color:rgb(255, 210, 132);
}</string>
         </property>
         <property name="text">
          <string>原神 [ PC端 ] 悬浮窗地图工具</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Page_2">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 font-family:&quot;微软雅黑&quot;;
	font-size:16px;
text-align: left;
	color:rgb(255, 255, 255);
}
QPushButton:hover{
   font-family:&quot;微软雅黑&quot;;
	font-size:16px;
	color:rgb(255, 210, 132);
}
QPushButton:pressed{
    font-family:&quot;微软雅黑&quot;;
	font-size:16px;
color:rgb(255, 210, 132);
}</string>
         </property>
         <property name="text">
          <string>天理启动器使用帮助 v0.8</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:rgba(0,0, 0,0);

}</string>
     </property>
     <attribute name="title">
      <string>更新</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>421</width>
        <height>66</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_Page_3">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 font-family:&quot;微软雅黑&quot;;
	font-size:16px;
text-align: left;
	color:rgb(255, 255, 255);
}
QPushButton:hover{
   font-family:&quot;微软雅黑&quot;;
	font-size:16px;
	color:rgb(255, 210, 132);
}
QPushButton:pressed{
    font-family:&quot;微软雅黑&quot;;
	font-size:16px;
color:rgb(255, 210, 132);
}</string>
         </property>
         <property name="text">
          <string>模组功能尚未实现，不必尝试</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Page_4">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 font-family:&quot;微软雅黑&quot;;
	font-size:16px;
text-align: left;
	color:rgb(255, 255, 255);
}
QPushButton:hover{
   font-family:&quot;微软雅黑&quot;;
	font-size:16px;
	color:rgb(255, 210, 132);
}
QPushButton:pressed{
    font-family:&quot;微软雅黑&quot;;
	font-size:16px;
color:rgb(255, 210, 132);
}</string>
         </property>
         <property name="text">
          <string>更新相关尚未实现，不必尝试</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:rgba(0,0, 0,0);

}</string>
     </property>
     <attribute name="title">
      <string>公告</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>421</width>
        <height>66</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_Page_5">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 font-family:&quot;微软雅黑&quot;;
	font-size:16px;
text-align: left;
	color:rgb(255, 255, 255);
}
QPushButton:hover{
   font-family:&quot;微软雅黑&quot;;
	font-size:16px;
	color:rgb(255, 210, 132);
}
QPushButton:pressed{
    font-family:&quot;微软雅黑&quot;;
	font-size:16px;
color:rgb(255, 210, 132);
}</string>
         </property>
         <property name="text">
          <string>测试交流群 QQ：538198823</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Page_6">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 font-family:&quot;微软雅黑&quot;;
	font-size:16px;
text-align: left;
	color:rgb(255, 255, 255);
}
QPushButton:hover{
   font-family:&quot;微软雅黑&quot;;
	font-size:16px;
	color:rgb(255, 210, 132);
}
QPushButton:pressed{
    font-family:&quot;微软雅黑&quot;;
	font-size:16px;
color:rgb(255, 210, 132);
}</string>
         </property>
         <property name="text">
          <string>侵删，emmm，开源代码，大家可以自己编译</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="GrayBackGround_TabPage_Rect">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>463</y>
      <width>460</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0, 0,100);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>235</y>
      <width>458</width>
      <height>211</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:0px solid rgb(248, 250, 251);
border-radius:0px;
background-image:url(:/page/resource/page/page1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>MainBackgroundImageRect</zorder>
   <zorder>TitleLabelRect</zorder>
   <zorder>pushButton_StartGame</zorder>
   <zorder>pushButton_TitleMin</zorder>
   <zorder>pushButton_TitleSet</zorder>
   <zorder>pushButton_TitleExit</zorder>
   <zorder>GrayBackGround_Right_Rect</zorder>
   <zorder>pushButton_Linke_1</zorder>
   <zorder>pushButton_Linke_2</zorder>
   <zorder>pushButton_Linke_3</zorder>
   <zorder>pushButton_Linke_4</zorder>
   <zorder>pushButton_Linke_5</zorder>
   <zorder>label_Page_Rect</zorder>
   <zorder>GrayBackGround_TabPage_Rect</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pushButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenshinImpactNaturalLaw.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_TitleMin</sender>
   <signal>clicked()</signal>
   <receiver>GenshinImpactNaturalLawClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1220</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>1344</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
